src/NuGet.Clients/NuGet.PackageManagement.UI/NuGet.PackageManagement.UI.csproj

```diff
@@ -117,9 +117,14 @@
     <Compile Include="Resources\Resources.xaml.cs">
       <DependentUpon>Resources.xaml</DependentUpon>
     </Compile>
+    <Compile Include="UserInterfaceService\DataStreamFromComStream.cs" />
     <Compile Include="UserInterfaceService\IModalProgressDialogSession.cs" />
     <Compile Include="UserInterfaceService\IUserSettingsManager.cs" />
     <Compile Include="Constants.cs" />
+    <Compile Include="UserInterfaceService\NuGetSettings.cs" />
```

Comment:
Afterthought: If those moved files are used in NuGet.PackageManagement.UI and NuGet.Tools, would it make sense to move them to NuGet.VisualStudio.Common ?

Reply:
tools refreences PMUI. so having that class in PMUI works fine.
why is it better to put one level further down?

Reply:
we talked offline

Reply:
Only INuGetUIFactory, SolutionUserOptions are referenced from NuGet.Tools. Other moved classes are referenced in NuGet.PackageManagement.UI